UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -381,6 +381,110 @@ public bool RemoveEmptys()
 				FieldsToRefresh.Clear();
 			}
 
+			if (Cut)
```

Comment:
Like this here could be extracted to a CutRegion() function.
So on review its just:
if(Cut == true) CutRegion();

Far easier to understand